Schemas.md

```diff
@@ -53,6 +53,9 @@ A schema is just a dictionary that takes the form described below:
     * `unicode_or_none`: Accepts data of type `unicode` or `None`. If the data is not `None`, it is normalized as if it were of type `unicode`.
     * `variable_keys_dict`: A dictionary with a variable number of keys and values.
 
+> [!IMPORTANT]  
+> When defining a schema with type `html`, it is mandatory to define `ui_config` with the key `rte_components` with value as one of the keys from [RTE_COMPONENT_CONFIGS](https://github.com/oppia/oppia/blob/ce64f781d35346cedd8584009a82fa5810ba7f6c/assets/constants.ts#L68) so that the rte displays as expected with the configured rte components.
```

Comment:
rte displays --> RTE displays

rte components --> RTE components

One question -- have you got lint validation to throw an error for this? It's easy to make a mistake here so I think we need to enforce it via code.